src/main/fc/config.c

```diff
@@ -168,6 +168,7 @@ static void activateConfig(void)
 
     useFailsafeConfig();
     setAccelerationTrims(&sensorTrims()->accZero);
+    setAccelerationFilter((float)accelerometerConfig()->acc_cut_hz);
```

Comment:
IMO there is no reason for cast here. How does it help with readability? (quality of generated code can be only worse)